### DISCUSSION TITLE
Folder / idea for naming of files for Collection with parent

### DISCUSSION THREAD

**Discussion by niloc7** [statamic/cms#4811] [Q&A]
So sorry for the complex title, I don't know how to word it out..

Basically trying to do this:

In COLLECTION -> PETS
persian-cat.md (parent)
 -> introduction.md (child)
 -> statistics.md (child)
 -> pictures.md (child)

And when I do the same now for bulldog.md with the same style of children pages (ie. introduction, statistics, pictures), then I would have a clash of names, and the file would be named pictures.2.md or something like that.

Example:
bulldog.md (parent)
 -> introduction.1.md (child)
 -> statistics.1.md (child)
 -> pictures.1.md (child)

Is there a way to not have the .1 or .23 .66? 

It's also not possible to put them in folders within folders (ie. persian-cat folder contains the child files)

What would be your suggestion for going about this guys? 

Thanks a lot! Been trying out for days.. :O

**Comment by jasonvarga** [comment]
What you're describing is how it works already. If you have two entries with the same slug, the second one will get saved with a suffix like `my-entry.123.md`

**Reply by niloc7** [reply]
Hei Jason, I think you might have misread what I wrote.. 

I'm asking if there is a way to NOT have the .1, .23, .66 instead of how to have it.. 

Thanks for the reply though!
